requirements.txt

```diff
@@ -1,3 +1,3 @@
 JSON-log-formatter
-Django
+Django<5.0
```

Comment:
Will cover that in the future. It requires installing different dependencies for lower Python versions.